docs/guides/editors.rst

```diff
@@ -371,3 +371,61 @@ Recommended
 
    Extremely powerful editing tool, best described with `this
    video. <http://emacsrocks.com/e13.html>`_ Package is ``multiple-cursors``.
+
+PyCharm
+-------
+
+PyCharm is a very powerful python IDE. However, it comes with a price - it's also quite resource-heavy.
+It has a `community edition <https://blog.jetbrains.com/pycharm/2013/10/pycharm-3-0-community-edition-source-code-now-available/>`_
+that is open-source since 2013. You can get it `here <https://www.jetbrains.com/pycharm/download/>`_.
+Some of its best out-of-the-box features are:
+
```

Comment:
**Required**
Please note what version of pycharm was used to write the examples that follow. In case the configuration/settings options move.